.github/workflows/stale.yml

```
@@ -0,0 +1,29 @@
+    - cron: '0 2 * * *' # every day at 02:00 UTC
+  workflow_dispatch:
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    permissions:
+      issues: write
+      pull-requests: write
+
+    steps:
+      - uses: actions/stale@v5
```

actionlint (1.7.10)

[error] the runner of "actions/stale@v5" action is too old to run on GitHub Actions. update the action's version to fix this issue

(action)